cogs/gatekeep/gatekeep.py

```diff
@@ -366,66 +404,66 @@ def check(msg: discord.Message):
     @user.command(name="add")
     @commands.guild_only()
     @checks.mod_or_permissions(administrator=True)
-    async def addUser(self, ctx: Context, id: int):
+    async def addUser(self, ctx: Context, user: discord.User):
         """Add a user to the watch list.
 
         Parameters:
         -----------
-        id: int
-            The user ID to be added to the watch list.
+        user: User
+            The user to be added to the watch list. This can be their username or ID.
         """
 
-        if id > 0:
+        if user.id > 0:
```

Comment:
Here and in removeUser, skip the `id > 0` check since we can rely on the ID to discord.User coercion to take care of this for us.